src/shippingservice/src/shipping_service/quote.rs

```diff
@@ -48,7 +50,10 @@ async fn request_quote(count: u32) -> Result<f64, Box<dyn std::error::Error>> {
     let mut reqbody = HashMap::new();
     reqbody.insert("numberOfItems", count);
 
-    let client = reqwest::Client::new();
+    let reqwest_client = reqwest::Client::new();
+    let client = ClientBuilder::new(reqwest_client)
```

Comment:
personal style choice, I would rewrite this as just:

```rust
let client = ClientBuilder::new(reqwest::Client::new())
```

Up to you.

Reply:
Thanks for the kind words. I guess I wanted to be explicit and transparent with the new dependency, but I also agree with the concise Rusty syntax. Will rewrite the bits as you propose.